README.md

```diff
@@ -32,7 +32,7 @@ Juliaup can be installed on Linux or Mac by executing
 curl -fsSL https://install.julialang.org | sh
 ```
 
-in a shell. Note that the Mac and Linux version are considered prerelease, have known bugs and might often break.
+in a shell. Although `juliaup` offers a reliable and seamless experience on Mac and Linux, these versions are still in development and thus are considered prerelease.
```

Comment:
```suggestion
in a shell.
```
I would just get rid of this comment altogether. Not clear to me in what meaningful sense the Linux and Mac versions are any less mature than the Windows version at this point.